RunningTheProduct.md

```diff
@@ -1,45 +1,44 @@
 ### Running the Product

```

Comment:
It really seems that this is the main heading, thus should be prefaced with a single # and that the others are subheadings and should be prefaced with ##